### DISCUSSION TITLE
Image is being reduced when engraving

### DISCUSSION THREAD

**Discussion by abody71** [arkypita/LaserGRBL#2411] [Q&A]
I am new to the laser world. 

I am setting up a home-built laser machine. I have engraved a .png and .svg file. Both images were about 1/3 of the original. Does anyone have any ideas about what is happening to reduce the engraving size?

Thanks
Allen

**Comment by Hostaler** [comment]
For example, make a 40x40mm square, measure it, if the result is not 40x40mm you will have to change some parameters.
These would be the parameters to change, but first share the result here and we will help you:
$100=250.000	X pasos/mm
$101=250.000	Y pasos/mm
$102=250.000	Pasos Z/mm
